scripts/rest_examples/reservation-create-aws.http

```diff
@@ -4,7 +4,7 @@ Content-Type: application/json
 X-Rh-Identity: {{identity}}
 
 {
-  "name": "amz-linux-us-east-1",
+  "names": ["amz-linux-us-east-1"],
```

Comment:
shall we revert this? :)

Reply:
Forgot this one, changed noop only. Fixed.